.github/workflows/release-discord-notify.yml

```
@@ -0,0 +1,27 @@
+name: release-discord-notify
+
+on:
+  release:
+    types: [published]
+
+jobs:
+  notify:
```

Name it as follows (we can have several targets for the notification):

```suggestion
  notify-discord:
```

---

@warstrolo can you please finalize? thx